visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -282,21 +282,20 @@ describe("codeMapMouseEventService", () => {
 			expect(threeSceneService.clearHighlight).toHaveBeenCalled()
 		})
 
-		it("should hover a node when no node is hovered, and an intersection was found and change the cursor to pointing", () => {
+		it("should hover a node when an intersection was found and the cursor is set to pointing", () => {
 			threeSceneService.getMapMesh = jest.fn().mockReturnValue({
 				checkMouseRayMeshIntersection: jest.fn().mockReturnValue(CODE_MAP_BUILDING)
 			})
-
 			threeSceneService.getHighlightedBuilding = jest.fn()
 
 			codeMapMouseEventService.updateHovering()
 
-			expect(threeSceneService.addBuildingToHighlightingList).toHaveBeenCalledWith(CODE_MAP_BUILDING)
+			expect(threeSceneService.addBuildingToHighlightingList).toHaveBeenCalled()
 			expect(threeSceneService.highlightBuildings).toHaveBeenCalled()
 			expect(document.body.style.cursor).toEqual(CursorType.Pointer)
 		})
 
-		it("should hover a node when no node is hovered, and an intersection was found but not change cursor in grabbing mode", () => {
+		it("should not hover node when an intersection was found and the cursor is set to grabbing", () => {
```

Comment:
Thanks for shortening the test names to make them more readable. I think we should replace the word "hover" with "highlight" That applies to the other test as well. What do you think?

Reply:
Sounds good, I will add that